src/OpenTelemetry.Extensions.Docker/CHANGELOG.md

```diff
@@ -1,12 +1,13 @@
-# Changelog - OpenTelemetry.Extensions.Docker
-
-This is the first release for the `OpenTelemetry.Extensions.Docker` project.
-The release targets
-[OpenTelemetry.Extensions.Docker](https://www.nuget.org/packages/OpenTelemetry.Extensions.Docker/).
-The project targets 1.2.0 of the [OpenTelemetry
-SDK](https://www.nuget.org/packages/OpenTelemetry/).
-
-The Docker extensions include plugin to extract resource detectors
-from docker environment (container id). For more details,
-please refer to the
-[README](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Extensions.Docker/README.md)
+# Changelog
+
+## Unreleased
+
+## 1.0.0-beta.1
+
+Released 2022-Jul-28
+
+* Targets 1.2.0 of the OpenTelemetry-SDK, net6.0 build added.
```

Comment:
It referenced `1.3.0`.

Reply:
Thank you, fixed. Copied it from old changelog initially.